src/metabase/upload.clj

```diff
@@ -170,19 +179,38 @@
     "unnamed_column"
     (u/slugify (str/trim raw-name))))
 
+(defn- is-pk?
+  [[col-name type]]
+  (and (#{"id" "pk"} (u/lower-case-en (name col-name)))
+       (isa? type ::int)))
+
+(defn- ->ordered-maps-with-pk-column
```

Comment:
What does `->` mean?

Reply:
"to". I've seen it used before in the same style as `inches->cm` but when you don't really care what the left-hand side is. If it were a public function I'd be more critical about the name, but since it's only used once the argument shape is pretty clear from context.

(and I wanted to capture that it's now doing more than the old name, `add-pk-column`, suggests)